packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -234,7 +234,7 @@ class BottomSheetCell extends Component {
 			// we show the TextInput just when the user wants to edit the value,
 			// and the Text component to display it.
 			// We also show the TextInput to display placeholder.
-			const shouldShowPlaceholder = isValueEditable && value === '';
+			const shouldShowPlaceholder = isInteractive && value === '';
```

Comment:
The `isInteractive` check not only checks whether `isValueEditable` is true, but also whether it's possible to interact with the cell. The thinking behind using this to determine whether to show a placeholder, is that a placeholder would likely only be set in cases where it's possible to interact with and change the value associated with a cell:

https://github.com/WordPress/gutenberg/blob/5176a3bb2cc7e8704538f1a18a532c6f98033c96/packages/components/src/mobile/bottom-sheet/cell.native.js#L168-L171

Reply:
As mentioned in the PR's description, this change might affect components that render `Cell` in the following scenario:
- Passing `placeholder` prop.
- Not passing `onChangeValue` (to not be considered `isValueEditable`).
- Passing `onPress` or `onLongPress` props.

I searched the codebase and looks like only the component `BottomSheetTextControl` matches, so this change seems safe.

P.S: These are the regex patterns I used in VSCode:
- `<BottomSheet.Cell([^>]|\n)+?onPress=([^>]|\n)+?placeholder=([^>]|\n)+?>`
- `<BottomSheet.Cell([^>]|\n)+?onLongPress=([^>]|\n)+?placeholder=([^>]|\n)+?>`
- `<BottomSheet.Cell([^>]|\n)+?placeholder=([^>]|\n)+?onPress=([^>]|\n)+?>`
- `<BottomSheet.Cell([^>]|\n)+?placeholder=([^>]|\n)+?onLongPress=([^>]|\n)+?>`